### PR TITLE
LPS-158124 BuildDockerImage task should depends on cleanDockerImage

### PR DESCRIPTION
**Pull request by Seiphon**

### CODE REVIEW

**Comment by Seiphon** [comment]
tests are passed here https://github.com/Seiphon/liferay-portal/pull/65

**Comment by gamerson** [comment]
hey guys, i'd like to discuss this change first before moving forward.

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java [hunk 1]
```diff
@@ -460,9 +471,15 @@ public void execute(Project p) {
 
 					setProperty.add(dockerImageId);
 
-					Property<String> property = dockerRemoveImage.getImageId();
+					Property<String> removeImageProperty =
```
Comment (petershin) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java:474]: dockerRemoveImageIdProperty

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java [hunk 2]
```diff
@@ -460,9 +471,15 @@ public void execute(Project p) {
...
+						dockerRemoveImage.getImageId();
+
+					removeImageProperty.set(dockerImageId);
+
+					Property<String> pruneImageProperty =
```
Comment (petershin) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java:479]: dockerPruneImageIdProperty

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/docker/DockerPruneImage.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+/**
+ * @author Seiphon Wang
+ */
+public class DockerPruneImage extends DockerExistingImage {
+
+	@Override
+	public void runRemoteCommand() {
+		Logger logger = getLogger();
+
+		Property<String> imageId = getImageId();
```
Comment (petershin) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/docker/DockerPruneImage.java:35]: imageIdProperty

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/docker/DockerPruneImage.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+		logger.quiet("Pruning image with ID \'" + imageId.get() + "\'.");
```
Comment (petershin) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/docker/DockerPruneImage.java:37]: if (logger.isQuietEnabled())

**Pull request closed without merging**